c2rust-analyze/src/dataflow/type_check.rs

```diff
@@ -484,5 +504,11 @@ pub fn visit<'tcx>(
         );
     }
 
+    for (&constant, const_lty) in &acx.const_tys {
+        let _ptr_id = const_lty.label;
+        let _expected_perms = const_perms(constant);
+        // TODO: check that perms match the expected ones
+    }
```

Comment:
How can I lookup the `PermissionSet` of a `PointerId` after dataflow typeck has run?

Reply:
That mapping is in the `Assignment`